src/gui/pages_menu/KM_GUIMenuMultiplayer.pas

```diff
@@ -560,9 +566,14 @@ procedure TKMMenuMultiplayer.MP_ServersDoubleClick(Sender: TObject);
 
 
 procedure TKMMenuMultiplayer.MP_HostClick(Sender: TObject);
+var
+  serverPortStr: string;
+  serverPort: Word;
 begin
   //Save the player and IP name so it is not lost if something fails
   MP_SaveSettings;
+  serverPortStr := IcsTrim(Edit_MP_ServerPort.Text);
+  serverPort    := atoi(serverPortStr);
```

Comment:
Is there a reason why you're using atoi instead of StrToInt? I didn't even know atoi existed in Pascal